l10n_br_account/models/account_invoice.py

```diff
@@ -43,6 +43,7 @@
 
 class AccountInvoice(models.Model):
     _inherit = 'account.invoice'
+    _order = "internal_number desc, id desc"
```

Comment:
Best way is to use date_invoice , because in account.invoice CAN be used to create other fiscal document types like NFS-e, using date_invoice, you can remove _order in l10n_br_account_product